metaflow/plugins/argo/argo_live_run.py

```diff
@@ -166,9 +164,9 @@ def _wait(self, wait_timeout) -> None:
                     f" minutes out of a possible {wait_timeout}"
```

Comment:
minor: missing a "minute" in the end. Maybe `minutes out of {wait_timeout} minutes limit.`